webapp/channels/src/packages/mattermost-redux/src/reducers/entities/users.ts

```diff
@@ -105,7 +105,7 @@ function removeProfileFromSet(state: RelationOneToMany<Team, UserProfile>, actio
 function currentUserId(state = '', action: GenericAction) {
     switch (action.type) {
     case UserTypes.RECEIVED_ME: {
-        const data = action.data || action.payload;
```

Comment:
`action.payload` is a remnant of an old mobile PR that seems to have been copied around randomly throughout this file. The only actions that have a `payload` field in the web app aren't related to user objects